_posts/news/2015-02-05-announcing-scalajs-0.6.0.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.0
+category: news
+tags: [releases]
+---
+{% include JB/setup %}
+
+We are thrilled to announce the final release of Scala.js 0.6.0!
+
+As of this version, we do not consider Scala.js to be experimental anymore.
+We believe it has reached maturity, and from now on, the language semantics as well as the APIs will only evolve in backward compatible ways, or go through proper deprecation cycles.
+
+Today is also the 2-year anniversary of Scala.js!
+The [first commit](https://github.com/scala-js/scala-js/commit/9ad7627c2418e5d345375705ca087a60e3aa2c22) was pushed on February 5, 2013.
+
+## Getting started
+
+If you are new to Scala.js, head over to
+[the tutorial]({{ BASE_PATH }}/doc/tutorial.html).
+
+## Release notes
+
+As the change in "major" version number witnesses, this release is *not* binary compatible with 0.5.x.
+Libraries need to be recompiled and republished using 0.6.0 to be compatible.
+More importantly, this release is not source compatible with 0.5.x either.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+The following libraries and testing frameworks have already been upgraded and published for 0.6.0:
+
+* [DOM types](https://github.com/scala-js/scala-js-dom): `"org.scala-js" %%% "scalajs-dom" % "0.7.0"`
+* [jQuery types](https://github.com/scala-js/scala-js-jquery): `"be.doeraene" %%% "scalajs-jquery" % "0.7.0"`
+* [scalajs-angulate](https://github.com/jokade/scalajs-angulate): `"biz.enef" %%% "scalajs-angulate" % "0.1"`
+
+**TODO Update this list**
+
+* [Scala.rx](https://github.com/lihaoyi/scala.rx): `"com.lihaoyi" %%% "scalarx" % "0.2.7-RC1"`
+* [ScalaTags](https://github.com/lihaoyi/scalatags): `"com.lihaoyi" %%% "scalatags" % "0.4.3-RC1"`
+* [Autowire](https://github.com/lihaoyi/autowire): `"com.lihaoyi" %%% "autowire" % "0.2.4-RC1"`
+* [µPickle](https://github.com/lihaoyi/upickle): `"com.lihaoyi" %%% "upickle" % "0.2.6-RC1"`
+* [µTest](https://github.com/lihaoyi/utest): `"com.lihaoyi" %%% "utest" % "0.2.5-RC1" % "test"`
+* [MiniTest](https://github.com/monifu/minitest): `"org.monifu" %%% "minitest" % "0.10" % "test"`
```

Comment:
I think we should not include this list into the announcement, since we should not update a blog post for content (but the list should be up-to-date...). Maybe we could just say that stuff is available (since people still have plenty of time to publish) and if necessary, keep a separate list on the site itself...